docs/git-comparison.md

```diff
@@ -134,6 +134,17 @@ parent.
       <td><code>jj diff -r &lt;revision&gt;</code></td>
       <td><code>git diff &lt;revision&gt;^ &lt;revision&gt;</code></td>
     </tr>
+    <tr>
+      <td>Show diff from one change to the current change</td>
```

Comment:
This could say "another change" instead of "one change".

Reply:
Done.